101-data-factory-v2-blob-to-postgresql-copy/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
doesn't look like you're actually using these - is that intentional?

Reply:
Ideally, we are supposed to use outputs from prereq 'AzureStorage_connectionString' and 'AzurePostgreSqlDatabase_connectionString'. But the PR check always fails with message - output should not contain secrets. So removed the outputs. 

The following ARM template does have secrets in output : 
https://github.com/Azure/azure-quickstart-templates/blob/master/101-data-factory-v2-blob-to-sql-copy/prereqs/prereq.azuredeploy.json

Reply:
@bmoore-msft : I have updated the ARM Templates using your suggestion to pass storage account resource id.